bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/handler/ThingLinkyRemoteHandler.java

```diff
@@ -0,0 +1,847 @@
+            updateState(LINKY_REMOTE_MAIN_GROUP, CHANNEL_USAGEPOINT_METER_ADDRESS_INSEE_CODE, UnDefType.UNDEF);
+            updateState(LINKY_REMOTE_MAIN_GROUP, CHANNEL_USAGEPOINT_METER_ADDRESS_STREET, UnDefType.UNDEF);
+
+            updateState(LINKY_REMOTE_MAIN_GROUP, CHANNEL_CONTACT_MAIL, UnDefType.UNDEF);
+            updateState(LINKY_REMOTE_MAIN_GROUP, CHANNEL_CONTACT_PHONE, UnDefType.UNDEF);
+
+        });
+    }
+
+    private synchronized @Nullable MetaData getMetaData() {
+        try {
+            EnedisHttpApi api = this.enedisApi;
+            MetaData result = new MetaData();
+            if (api != null) {
+                if (supportNewApiFormat()) {
+                    result.identity = api.getIdentity(this, config.prmId);
+                    result.contact = api.getContact(this, config.prmId);
+                    result.contract = api.getContract(this, config.prmId);
+                    result.usagePoint = api.getUsagePoint(this, config.prmId);
+                } else {
+                    UserInfo userInfo = api.getUserInfo(this);
+                    PrmInfo prmInfo = api.getPrmInfo(this, userInfo.userProperties.internId, config.prmId);
+                    PrmDetail details = api.getPrmDetails(this, userInfo.userProperties.internId, prmInfo.idPrm);
+
+                    result.identity = Identity.convertFromUserInfo(userInfo);
+                    result.contact = Contact.convertFromUserInfo(userInfo);
+                    result.contract = Contract.convertFromPrmDetail(details);
+                    result.usagePoint = UsagePoint.convertFromPrmDetail(prmInfo, details);
+                }
+            }
+            return result;
+        } catch (LinkyException e) {
+            logger.error("Exception occurs during data update for {} : {}", config.prmId, e.getMessage(), e);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        }
+
+        return null;
+    }
+
+    /**
+     * Request new data and updates channels
+     */
+    private synchronized void updateData() {
+        // If one of the cache is expired, force also a metaData refresh to prevent 500 error from Enedis servers !
+        logger.info("updateData() called");
+        logger.info("Cache state {} {} {}", dailyConsumption.isPresent(), dailyConsumptionMaxPower.isPresent(),
+                loadCurveConsumption.isPresent());
+
+        if (!dailyConsumption.isPresent() || !dailyConsumptionMaxPower.isPresent()
+                || !loadCurveConsumption.isPresent()) {
+            logger.info("invalidate metaData cache to force refresh");
+            metaData.invalidate();
+        }
+
+        logger.info("updateMetaData() called");
+        updateMetaData();
+        logger.info("updateEnergyData() called");
+        updateEnergyData();
+        logger.info("updatePowerData() called");
+        updatePowerData();
+        logger.info("updateLoadCurveData() called");
```

Comment:
Don't do this type of logging.
```suggestion
        updateMetaData();
        updateEnergyData();
        updatePowerData();
```